.gitpod.yml

```diff
@@ -0,0 +1,5 @@
+ports:
+- port: 4242
+tasks:
+- init: yarn install
+  command: PERFHTML_HOST="0.0.0.0" yarn start
```

Comment:
Do you know if we can have a second terminal opened, given this terminal will stay busy with the dev server?

Reply:
I see we can have several `command` lines that each will open a new terminal: is an empty `command` property legal? Otherwise just an `echo` will work :)

Also maybe adding a "echo" before each command to explain what we're doing could be nice.

Reply:
Interesting suggestion, thanks. You can already open new terminals from the UI (`Terminal` > `New Terminal`), but we can indeed automatically open a new terminal to allow newcomers to run commands without having to kill the dev server.